packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -362,4 +362,29 @@
         @background="#eee"
       /></T.Panel>
   </Hds::Tabs>
+
+  <h5 class="dummy-h6">With overflowing tabs</h5>
+  <Hds::Tabs as |T|>
+    <T.Tab>One one-thousand</T.Tab>
+    <T.Tab>Two one-thousand</T.Tab>
+    <T.Tab>Three one-thousand</T.Tab>
+    <T.Tab>Four one-thousand</T.Tab>
+    <T.Tab>Five one-thousand</T.Tab>
+    <T.Tab>Six one-thousand</T.Tab>
+    <T.Tab>Seven one-thousand</T.Tab>
+    <T.Tab>Eight one-thousand</T.Tab>
+    <T.Tab>Nine one-thousand</T.Tab>
+    <T.Tab>Ten one-thousand</T.Tab>
+
+    <T.Panel>Content #1</T.Panel>
+    <T.Panel>Content #2</T.Panel>
+    <T.Panel>Content #3</T.Panel>
+    <T.Panel>Content #4</T.Panel>
+    <T.Panel>Content #5</T.Panel>
+    <T.Panel>Content #6</T.Panel>
+    <T.Panel>Content #7</T.Panel>
+    <T.Panel>Content #8</T.Panel>
+    <T.Panel>Content #9</T.Panel>
+    <T.Panel>Content #10</T.Panel>
```

Comment:
Would it be possible to use the same approach as the examples before, using the `<DummyPlaceholder>`?